torch/fx/quantization.py

```diff
@@ -0,0 +1,329 @@
+r'''
```

Comment:
We should put this in the test folder similar to `test_jit.py` that has its own subfile. Otherwise someone might see it and think it has to do with real quanitization.